app/services/application/ApplicationService.scala

```diff
@@ -133,6 +133,7 @@ trait ApplicationService extends EventSink with CurrentSchemeStatusHelper {
       }
   }
 
+  //scalastyle:off
```

Comment:
What was scalastyle complaining about here? can you turn off the specific rule e.g. // scalastyle:off method.length

Reply:
also complaining about cyclomatic complexity